domains/Accounts/Controllers/VerifyEmailController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Accounts\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+
+class VerifyEmailController extends Controller
+{
+    protected ?User $user = null;
+
+    public function __invoke(Request $request): Response
```

Comment:
Note that the user will land here after clicking a link in the email and will need visual feedback to know if everything went well or not. This method should return a blade view with that information.